src/main/ruby/core/marshal.rb

```diff
@@ -298,18 +298,12 @@ class String
   end
 end
 
-class Fixnum
```

Comment:
Does MRI retain the old classes so old data can be unmarshalled (e.g., serialized to disk or a DB)? If so, we should retain that as well to make migrating to TruffleRuby easier.

Reply:
`serialize_integer` handles both Fixnum and Bignum, so this is functionally equivalent and produces the same bytes as before.